lib/core/src/model.rs

```diff
@@ -806,17 +828,9 @@ impl ReceiveSwap {
 pub struct RefundableSwap {
     pub swap_address: String,
     pub timestamp: u32,
+    /// Amount that is refundable, which is different than the swap amount
```

Comment:
It *might* be the same right? So maybe its clearer to write
```suggestion
    /// Amount that is refundable, from all UTXOs
```

Reply:
Good point, done: 723679ea0a2bd6acec6be71b420c9e1313c25ee5